tests/testthat/identify-roxygen-examples/1-one-function-example-last-proper-run.R

```diff
@@ -3,5 +3,5 @@
 #' Performs various substitutions in all `.R` files in a package
 #' (code and tests).
 #' Carefully examine the results after running this function!
-#' @example style_pkg(style = tidyverse_style, strict = TRUE)
+#' @examples style_pkg(style = tidyverse_style, strict = TRUE)
```

Comment:
Maybe just to test the extra spaces, I think we could remove a space here to make sure regex works.

Reply:
the space before `@`?

Reply:
Oops. Haha ok - now I see what you mean.